src/bloqade/squin/groups.py

```diff
@@ -44,3 +44,11 @@ def run_pass(method):
         py_mult_to_mult_pass(method)
 
     return run_pass
+
+
+@ir.dialect_group(kernel.add(noise.dialect))
```

Comment:
maybe just put noise dialect inside kernel? like how to define `qasm2.extended`?

Reply:
Yes, this was just because I had a separate decorator in #273 with an additional rewrite. I'll add it to the kernel directly. But first, I'll need to see whether we're going with #273 or this version.